doc/syscalls/00008_low_level_debug.md

```diff
@@ -0,0 +1,74 @@
+---
+driver number: 0x00008
+---
+
+# Low-Level Debug
+
+## Overview
+
+The low-level debug driver provides tools to diagnose userspace issues that make
+normal debugging workflows (e.g. printing to the console) difficult. It allows
+apps to print status codes and numeric information using only the `command`
+system call, and is easy to call from handwritten assembly. The driver is in
+capsules/src/low\_level\_debug.rs.
+
+## Command
+
+  * Description: command() is used to print status codes and numbers. The driver
+    does not provide a way for an app to wait for the print completed. If the
+    app prints too many messages in a row, the driver will print a message
+    indicating it has dropped some debug messages.
+
+  * ### Command Number: 0
+
+    **Description**: Driver check.
+
+    **Argument 1**: Unused
+
+    **Argument 2**: Unused
+
+    **Returns**: SUCCESS
+
+  * ### Command Number: 1
+
+    **Description**: Print a predefined status code. The available status codes
+    are listed later in this doc.
+
+    **Argument 1**: Status code to print
+
+    **Argument 2**: Unused
+
+    **Returns**: SUCCESS
+
+  * ### Command Number: 2
+
+    **Description**: Print a single number. The number will be printed in
+    hexadecimal.
+
+    **Argument 1**: Number to print
+
+    **Argument 2**: Unused
+
+    **Returns**: SUCCESS
+
+  * ### Command Number: 3
+
+    **Description**: Print two numbers. The numbers will be printed in
+    hexadecimal.
+
+    **Argument 1**: First number to print
+
+    **Argument 2**: Second number to print
+
+    **Returns**: SUCCESS
```

Comment:
I don't see the use case here. The alert codes (previously "status codes") are meant to be used in libraries, and are predefined to prevent collisions between projects. Having a number in addition to an alert code would re-introduce the collision problem.

At the application level, or for temporary debugging, using the "print 2" command to print a location ID + value makes a lot of sense, and is extremely useful in practice.

I added some explanations to the document, does that help?

Reply:
Yes! Thank you 👍